src/authors/components/AuthorResultItem.jsx

```diff
@@ -1,29 +1,38 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
-import { Map } from 'immutable';
+import { Map, List } from 'immutable';
 
 import ResultItem from '../../common/components/ResultItem';
 import AuthorName from './AuthorName';
 import ArxivCategoryList from './ArxivCategoryList';
 import ExperimentList from './ExperimentList';
+import AuthorAffiliationList from '../../common/components/AuthorAffiliationList';
 
 class AuthorResultItem extends Component {
   render() {
     const { metadata } = this.props;
 
     const name = metadata.get('name');
     const recordId = metadata.get('control_number');
-    const institution = metadata.getIn(['positions', 0, 'institution']);
+    const currentPositions = metadata
+      .get('positions', List())
+      .filter(position => position.get('current'))
+      .map(position => position.get('institution'));
```

Comment:
this logic can be moved to a function in a new file called `utils.js` (similar to `common/utils.js`) under `authors`. And you can write few tests.

Something like `getCurrentAffiliationsFromPositions(positions)`